package-lock.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "ieee-ghost",
+  "version": "1.0.0",
+  "lockfileVersion": 1
+}
```

Comment:
Looks like you accidentally ran `npm install` within the main directory. Please remove this file.

Reply:
hmm, I recommitted but the commit isn't showing up. should I just open a new request?

Reply:
Did you push the commit?

Reply:
It looks like you commit on the same branch that I already merged into master, please make commits on a new branch that's based off master (not a stale branch).